docs/tutorials/essentials/part-2-app-structure.md

```diff
@@ -514,7 +514,7 @@ There's one more function in this file, but we'll talk about that in a minute wh
 
 :::info Want to Know More?
 
-See [the Redux Thunk docs](https://github.com/reduxjs/redux-thunk), the post [What the heck is a thunk?](https://daveceddia.com/what-is-a-thunk/) and the [Redux FAQ entry on "why do we use middleware for async?"](../../faq/Actions.md#how-can-i-represent-side-effects-such-as-ajax-calls-why-do-we-need-things-like-action-creators-thunks-and-middleware-to-do-async-behavior) for more information.
+See [the Redux Thunk docs](https://redux.js.org/usage/writing-logic-thunks), the post [What the heck is a thunk?](https://daveceddia.com/what-is-a-thunk/) and the [Redux FAQ entry on "why do we use middleware for async?"](../../faq/Actions.md#how-can-i-represent-side-effects-such-as-ajax-calls-why-do-we-need-things-like-action-creators-thunks-and-middleware-to-do-async-behavior) for more information.
```

Comment:
:hand: Thanks! Same-site doc links need to point to the relevant Markdown file instead of the public URL - can you do that here?

Reply:
done @markerikson